server/lib/video-transcoding.ts

```diff
@@ -230,11 +231,13 @@ export {
 async function onVideoFileTranscoding (video: MVideoWithFile, videoFile: MVideoFile, transcodingPath: string, outputPath: string) {
   const stats = await stat(transcodingPath)
   const fps = await getVideoFileFPS(transcodingPath)
+  const metadata = await getMetadataFromFile(transcodingPath)
 
   await move(transcodingPath, outputPath)
 
   videoFile.size = stats.size
   videoFile.fps = fps
+  videoFile.metadata = metadata
```

Comment:
please select the field you want to include in the json, it's better for security and safety